Classes/common/touchdb/TDPuller.m

```diff
@@ -83,7 +83,10 @@ - (void)beginReplicating
 - (void)startChangeTracker
 {
     Assert(!_changeTracker);
-    TDChangeTrackerMode mode = (_continuous && _caughtUp) ? kLongPoll : kOneShot;
+    //continuous / longpoll modes are not supported or available at the CDT* level.
+    //As such, the new TDURLConnectionChangeTracker also only supports one-shot query
+    //to the _changes feed. 
+    TDChangeTrackerMode mode = kOneShot;
```

Comment:
I suggest removing the constants for long poll and continuous as we're not supporting them right  now,